indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -107,11 +110,35 @@ def _process(self):
 
 
 class RHCreateAbstract(RHAbstractListBase):
+    """Create an abstract from scratch or with existing cloned data."""
+
+    def clone_fields(self, abstract):
+        field_names = ['title', 'description', 'submission_comment', 'submitted_for_tracks', 'submitted_contrib_type']
+        field_data = {f: getattr(abstract, f) for f in field_names}
+        person_links = []
+        link_attrs = get_simple_column_attrs(AbstractPersonLink)
+        for old_link in abstract.person_links:
+            link = AbstractPersonLink(person=old_link.person)
+            link.populate_from_attrs(old_link, link_attrs)
+            person_links.append(link)
+        field_data['person_links'] = person_links
+        for f in abstract.field_values:
+            field_data[f'custom_{f.contribution_field_id}'] = f.data
+        return field_data
+
+    @use_kwargs({'abstract_id': fields.Int()}, location='query')
+    def _process_args(self, abstract_id=None):
```

Comment:
```suggestion
    @use_kwargs({
        'abstract_id': fields.Int(load_default=None)
    }, location='query')
    def _process_args(self, abstract_id):
```

We usually use `load_default` (formerly `missing`) instead of setting a default value on the kwarg. No real difference, just for consistency